service/build.gradle

```diff
@@ -13,6 +13,11 @@ apply plugin: 'org.springframework.boot'
 apply plugin: 'checkstyle'
 apply plugin: "com.github.spotbugs"
 
+java {
+    sourceCompatibility = JavaVersion.VERSION_21
+    targetCompatibility = JavaVersion.VERSION_21
+}
```

Comment:
You'll have to explain what these do. They weren't there before, so I can only assume they're important?

Reply:
this is just to make sure that we are using Java 21 in case when there are any other Java versions installed on any env where this project is run. It is just to avoid ambiguity